spec/node/grunt/helper-spec.js

```diff
@@ -29,6 +29,9 @@ describe('grunt helper', function () {
 
             beakerTasks = ['beaker-task-1', 'grunt-cli', 'beaker-task-2'];
             localTasks = ['local-task-1', 'local-task-2'];
+            grunt.option = function () {
+                return null;
+            };
```

Comment:
In line 25, a grunt mock is created.

>grunt = jasmine.createSpyObj('grunt', ['loadNpmTasks', 'initConfig', 'registerTask'])

You can add `option` to this list instead of assigning a function manually.